lib/xcknife/test_dumper.rb

```diff
@@ -299,7 +299,12 @@ def simctl
     end
 
     def gtimeout
-      "gtimeout -k 5 #{@simctl_timeout}"
+      return nil if gtimeout_path.empty?
```

Comment:
If someone specified a timeout but `gtimeout` isn't installed, would this fail later? If so, maybe good if we dumped out a short usage error explaining the dependency requirement?

Reply:
Good point - It won't fail, but I'm missing some error output. That could be frustrating to work through... fixed in b183cdc